src/components/interactive-builder/add-question.modal.tsx

```diff
@@ -238,7 +238,7 @@ const AddQuestionModal: React.FC<AddQuestionModalProps> = ({
             selectedConcept?.allowDecimal === false && { disallowDecimals: true }),
           ...(questionType === 'programState' && {
             answers: selectedProgramState.map((answer) => ({
-              value: answer.concept.uuid,
+              value: answer.uuid,
```

Comment:
Should the `answer` object align with [this answer interface](https://github.com/openmrs/openmrs-esm-form-builder/blob/8426207598bc6cab16a84279839a2f1d28ac20f9/src/types.ts#L150), or should the interface be updated to reflect some of the additions/omissions?

Reply:
The value for this specific question is the state uuid and not the concept uuid, i think its more of [this](https://github.com/openmrs/openmrs-esm-form-builder/blob/8426207598bc6cab16a84279839a2f1d28ac20f9/src/types.ts#L166C18-L166C31)

Reply:
Not quite, the type `ProgramState` contains a concept object and a uuid